src/azure-cli/azure/cli/command_modules/acr/custom.py

```diff
@@ -258,6 +258,8 @@ def acr_login(cmd,
                        'docker commands, to avoid authentication errors, use all lowercase.')
 
     from subprocess import PIPE, Popen
+    logger.debug("Invoking '%s --username %s --password <redacted> %s'",
```

Comment:
will there be user sensitive data in docker_command?

Reply:
@yungezz, i am using `<redacted>` to avoid output the token, hence no worries here. Please go ahead to merge. Thanks